### DISCUSSION TITLE
Roadmap Q3 2022

### DISCUSSION THREAD

**Discussion by msujew** [eclipse-langium/langium#573] [Announcements]
Here's a list of what the contributors of Langium plan to do for the next 3 months, with some more probably still to come. At the end of Q3 we will likely have another release (0.5)

* [x] @dhuebner Configuration provider, already started at https://github.com/langium/langium/pull/519
* [ ] @montymxb @Lotes @pluralia Improve type inference and validation using formal methods (will be moved to Q4)
* [x] @msujew @gfontorbe More out-of-the-box support for LSP services https://github.com/langium/langium/issues/595
* [x] @msujew @kaisalmen Langium browser support
* [x] @montymxb [Monarch syntax highlighting](https://microsoft.github.io/monaco-editor/monarch.html) generator for monaco users. https://github.com/langium/langium/issues/582

**Comment by spoenemann** [comment]
See also the [v0.5.0 Milestone](https://github.com/langium/langium/milestone/4) for assigned issues & PRs. But note that not every issue assigned to that milestone will actually be solved; we tend to move a bulk of issues to the following milestone when the more important things are complete and it's time for a release.
